main.js

```diff
@@ -0,0 +1,86 @@
+const howSec = document.querySelector("#push");
+const newNumber = document.querySelector("#new__task input");
+
+howSec.addEventListener("click", function () {
+  if (newNumber.value) {
+    const newTask = document.createElement("span");
+    newTask.classList.add("new__task__to__do");
```

Comment:
`new__task__to__do` - seems not corresponding for current task

```suggestion
    span.classList.add("action");
```